resources/install.rb

```diff
@@ -21,26 +20,34 @@
 property :security_mode, String, equal_to: ['Windows Authentication', 'Mixed Mode Authentication'], default: 'Windows Authentication'
 property :sa_password, String
 property :sysadmins, [Array, String], default: ['Administrator']
-property :agent_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :agent_startup, String, equal_to: ['Automatic', 'Manual', 'Disabled', 'Automatic (Delayed Start)'], default: 'Disabled'
+property :agent_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :agent_account_pwd, String
+property :ft_account, String, default: 'NT Service\MSSQLFDLauncher'
+property :ft_account_pwd, String
 property :rs_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :rs_account_pwd, String
 property :rs_startup, String, equal_to: ['Automatic', 'Manual', 'Disabled', 'Automatic (Delayed Start)'], default: 'Automatic'
 property :rs_mode, String, default: 'FilesOnlyMode'
 property :as_sysadmins, [Array, String], default: ['Administrator']
 property :sql_account, String, default: 'NT AUTHORITY\NETWORK SERVICE'
 property :sql_account_pwd, String
+property :sql_instant_file_int, [true, false], default: false
+property :is_account, String, default: 'NT AUTHORITY\NetworkService'
+property :is_account_pwd, String
+property :is_startup, String, default: 'Automatic'
 property :browser_startup, String, equal_to: ['Automatic', 'Manual', 'Disabled', 'Automatic (Delayed Start)'], default: 'Disabled'
 property :version, [Integer, String], default: '2012'
 property :source_url, String
 property :package_name, String
 property :package_checksum, String
 property :installer_timeout, Integer, default: 1500
 property :accept_eula, [true, false], default: false
+property :suppress_privacy_statement, [true, false], default: true
 property :product_key, String
 property :update_enabled, [true, false], default: true
 property :update_source, String, default: 'MU'
+property :use_microsoft_update, [true, false], default: false
```

Comment:
I think the default needs to be true since the default for update_source is MU which is Microsoft update